packages/datadog_webview_tracking/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: datadog_webview_tracking
 description: A package for tracking Datadog sessions in a webview
-version: 1.0.0-beta.3
+version: 1.1.0
```

Shouldn't it be just `1.0.0` here?

```suggestion
version: 1.0.0
```

---

No, develop always has the next version of the library. The release branch has the current version.